docs/README.md

```diff
@@ -2,20 +2,21 @@
 
 IoT Identity Service is made up of three services that provide provisioning and cryptographic services:
 
-- [Identity Service](identity-service.md)
+- [Identity Service](identity-service.html)
 
-    This service provisions the device identity and module identities with Azure. 
+    This service provisions the device's [identity](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry) with Azure. The device identity can be based on symmetric keys or X.509 certificates, and be used with manual device registrations, DPS individual enrollments or DPS group enrollments.
 
-    The Identity Service provides access to host-level processes and container modules to connect to the cloud using tokens or X.509 certificate credentials, corresponding to their respective identities.
+    The Identity Service also provides access to native processes ("host processes") to connect to the cloud. Depending on how the user configures the Identity Service, host processes can get their own *module identities* provisioned by the Identity Service, or use the device identity, or both.
 
-    Note that this service performs the entirety of provisioning.
+    The Identity Service also provisions module identities for containerized modules, ie IoT Edge modules. However container modules do not communicate with the Identity Service directly; instead the provisioning of their identities is handled by the IoT Edge runtime on the modules' behalf. Refer to <https://github.com/Azure/iotedge> for more details about the IoT Edge runtime.
```

Comment:
Given we purposely separating this repo out for both edge and non-edge devices, it would be helpful to clarify what is generic.

Reply:
That's what I've already done. The previous paragraph talks about host processes. This one talks about IoT Edge modules. What do you want changed?

Reply:
Yes, I saw that. Thanks. 

How about this: 
"IoT Identity Service is made up of three services that provide provisioning and cryptographic services for both edge and non-edge devices:"

"Specially for edge devices, the identity service also provision identities for containerized modules. ... "

Reply:
Okay, I've done that, and also moved it to its own section to be clearer how the MR talks to all three.